apps/admin_app/config/config.exs

```diff
@@ -16,7 +16,9 @@ config :admin_app, AdminAppWeb.Endpoint,
   pubsub: [name: AdminApp.PubSub, adapter: Phoenix.PubSub.PG2],
   token_maximum_age: System.get_env("TOKEN_MAXIMUM_AGE"),
   sendgrid_sender_mail: System.get_env("SENDGRID_SENDER_EMAIL"),
-  password_reset_salt: System.get_env("PASSWORD_RESET_SALT")
+  password_reset_salt: System.get_env("PASSWORD_RESET_SALT"),
+  support_url: "https://admin.aviacommerce.org",
```

Comment:
Use `System.get_env` here and add use the key from env file if present otherwise, add the keys.